### PR TITLE
Updated lodash to 4 version

### PR DESCRIPTION
**Pull request by AleshaOleg**

### CODE REVIEW

File: src/components/tooltip/HoverTooltip.js [hunk 1]
```diff
@@ -80,7 +79,9 @@ export default class HoverTooltip extends PureComponent {
       <div className={classes[`wrapper${inline ? 'Inline' : ''}`]}>
         <span
           onMouseOver={this.onMouseOver}
+          onFocus={this.onMouseOver}
           onMouseOut={this.onMouseOut}
+          onBlur={this.onMouseOut}
```
Comment (nikgraf) [inline src/components/tooltip/HoverTooltip.js]: Why this change?

File: src/components/tags-input/TagsInput.js [hunk 2]
```diff
@@ -91,6 +90,10 @@ export default class TagsInput extends PureComponent {
     this.props.deleteTag(item)
   }
 
+  clearFilter() {
```
Comment (nikgraf) [inline src/components/tags-input/TagsInput.js]: What's the reason for extracting it to a separate function?

Comment (AleshaOleg) [reply src/components/tags-input/TagsInput.js]: The problem was with usage of redundant `shouldComponentUpdate` (no redundant usage component update). Firstly I fixed it with replacing `PureComponent` with `Component`, but then new error appeared - https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-will-update-set-state.md. And I decided to save `PureComponent` and extracted setting new state to new function :D

File: src/components/tooltip/HoverTooltip.js (hunk 1 above)
Comment (AleshaOleg) [reply src/components/tooltip/HoverTooltip.js]: I added onBlur and onFocus in order to remove the error about https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/mouse-events-have-key-events.md.
